### ISSUE TITLE
How to get the same colors as depicted in the screenshot

### ISSUE THREAD

**Issue by Jones-S** [halfo/lambda-zsh-theme#4]
Hi would like to use the theme in the way you are. I realize that I have to change colors in the Profile Manager of the terminal itself, but I can't quite figure out how to change the lamda sign for example. And which color belong to which kind of message.

Maybe you could get me a hint on that?
Thanks anyway.

Cheers 

![screen shot 2016-02-23 at 18 24 40](https://cloud.githubusercontent.com/assets/7059580/13260200/c130b35c-da5a-11e5-85dd-759f8eda1023.png)

**Comment by halfo** [comment]
Hey! I think this #2  is a similar issue. If that doesn't solve your problem then please let me know again.
Cheers

**Comment by Jones-S** [comment]
Thanks for the answer.
I tried a lot of things including converting an .itermcolors file to a .terminal file, which I could import into the terminal profile manager. But no luck so far. All the scripts I found are probably out of date or somehow not working... (.e.g https://gist.github.com/bitserf/365fae61b96167fe6402)

Also all the other color-scheme-files do not cover the necessary ones for the terminal. One can have sublime, xcode, atom, notepad.... almost anything. But I don't find a way to use it in my terminal.

Maybe you could export a colors.terminal file? Or do you have any other thought about that?
Thanks in advance

**Comment by Jones-S** [comment]
Oh no: I found this:
https://github.com/idleberg/Paraiso-Terminal
obviously he did not link that file in the the post you showed me. Anyway thank you. It worked.
Only one thing left: in your screenshot you have this beautiful lambda sign also colored. Yours looks much more colorful anyway. How did you do that?

![screen shot 2016-02-24 at 12 17 08](https://cloud.githubusercontent.com/assets/7059580/13283951/4c672820-daf1-11e5-8dc6-e928fd3dc32f.png)

**Comment by halfo** [comment]
I'm using gnome terminal on Ubuntu. I'm not sure if my settings would work on Mac. But I can share that along with a screenshot of the color settings. That's the best I can do right now (don't have a Mac :( ). Though you've to wait one more day for it as I don't have that working station with me right now. Sorry about that.

**Comment by Jones-S** [comment]
No worries. If you could do that, that would be awesome. If not, I won't die :smile: 
Thanks for all the help! :v:

**Comment by halfo** [comment]
Sorry for the delay. [Here](https://github.com/halfo/lambda-mod-zsh-theme/files/161437/paraiso-dark.txt) is my configuration file for Paraiso theme. If it doesn't work then you've to set the colors manually. Here is the screenshot of my profile colors.

![profile-colors](https://cloud.githubusercontent.com/assets/2076914/13572032/4e884840-e4a2-11e5-852c-e86a22383025.png)

**Comment by Jones-S** [comment]
The settings for the palette is a little different from the mac terminal settings. but anyway. thanks a lot for the help here. I got it more or less together. :+1:

**Comment by halfo** [comment]
That's awesome!!! You may put it on github so someone else can get help from that.

**Comment by Jones-S** [comment]
Hm I guess you think it's more than it actually is. I downloaded the terminal settings from https://github.com/idleberg/Paraiso-Terminal and used that. Additionally I change some colors which I sampled from your screenshot. But that's about it. I don't have a colored lamda, I don't have that arrow. I just more or less have the colors but otherwise I gave up. So I guess my terminal settings would not help anyone much...
